lib/ReactViews/Map/Panels/HelpPanel/HelpVideoPanel.jsx

```diff
@@ -111,13 +118,27 @@ class HelpVideoPanel extends React.Component {
               <Icon glyph={Icon.GLYPHS.play} />
             </button>
           </div>
-          <Spacing bottom={3} />
-          <Text bold heading>
+          <Spacing bottom={5} />
+          <Text
+            css={`
+              color: #404855;
+              font-size: 23px;
+              line-height: 31px;
```

Comment:
Propify this combination of font-size & line-height (looks like it sits between the existing props `large` & `heading`)

Reply:
Made it a `subHeading` for now, since it's quite a fair bit bigger than `extraLarge` (what I would've otherwise named it). 

Are you still planning on bringing the h1-h6 headings, or something along those lines, from terrace that you mentioned a while back? 🤔